app/models/task.rb

```diff
@@ -60,7 +57,7 @@ def update_status(state, logged_at)
 
     with_lock do
       # see if there is already another status that came in that was logged_at > than this one.
-      future_log = task_logs.where(status: Task.statuses.keys).where('logged_at > ?', logged_at).exists?
+      future_log = task_logs.where(status: Task.statuses.values).where('logged_at > ?', logged_at).exists?
```

Comment:
Line is too long. [106/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.